docs/docs/build/modules/02-provider.md

```diff
@@ -1679,8 +1687,9 @@ where `update-consumer-msg.json` contains:
       "validator_set_cap": 50,
       "allowlist":["cosmosvalcons1l9qq4m300z8c5ez86ak2mp8znftewkwgjlxh88"],
       "denylist":[],
-      "min_stake": 1000,
-      "allow_inactive_vals":true
+      "min_stake": "1000",
```

Comment:
Is the min_stake a string or a number? Why is it between `"`?

Reply:
It was changed during the merge with main, but those changes should be reverted. It's been fixed now.